.github/workflows/add-to-project.yml

```diff
@@ -0,0 +1,14 @@
+name: Apply project management flow
+on:
+  issues:
+    types: [opened]
+  pull_request_target:
+    types: [opened]
+jobs:
+  project-management:
+    runs-on: ubuntu-latest
+    name: Apply project management flow
+    steps:
+      - uses: honeycombio/oss-management-actions/projects@v1
```

Comment:
At some point we're gonna need to get a little more sophisticated in this action I think, since there will be teams who manage repos differently. But we can cross that bridge when we get there.